<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-4IMJ5PC" failures="1" tests="5" name="Personal Loan" time="0.236" errors="0" timestamp="2023-04-25T17:04:36 IST">
  <testcase classname="test.day1" name="demo" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.demo(day1.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="test.day2" name="ploan" time="0.016"/>
  <testcase classname="test.day4" name="LoginAPIHomeLoan" time="0.041"/>
  <testcase classname="test.day4" name="MobileLoginHomeLoan" time="0.002"/>
  <testcase classname="test.day4" name="WebLoginHomeLoan" time="0.001"/>
  <testcase classname="test.day1" name="secondTest" time="0.0">
    <ignored/>
  </testcase> <!-- secondTest -->
</testsuite> <!-- Personal Loan -->
